catalyst/data/sampler.py

```diff
@@ -195,6 +195,100 @@ def __iter__(self) -> Iterator[int]:
         return iter(inds)
 
 
+class DynamicBalanceClassSampler(Sampler):
```

Comment:
hi, @Dokholyan 
Could you please provide a small example for this `DynamicBalanceClassSampler` usage?
for example, like here - https://github.com/catalyst-team/catalyst/blob/master/catalyst/data/sampler.py#L306L325

Reply:
btw, could you also please add this class to the docs? here - https://github.com/catalyst-team/catalyst/blob/master/docs/api/data.rst#samplers
but please keep the alphabetical order ;)